docs/products/access/api-reference/backchannel-custom-registration.md

```diff
@@ -0,0 +1,64 @@
+# Custom Registration Backchannel Communication API
```

Comment:
We have not removed it yet because there is no alternative available. We can deploy an Extension Engine per tenant and then it still works.

Reply:
It is still accurate because we decided not to remove it as long as there is no replacement. Most likely the feature will be there in the future, but the execution of the script will change.